package.json

```diff
@@ -162,6 +166,10 @@
                     "command": "sfca.runDfaOnSelectedMethod",
                     "when": "false"
                 },
+                {
+                    "command": "sfca.runDfa",
+                    "when": "false"
```

Comment:
I'm setting this as false so the new command will not show up until Delta run is fully complete. To test this locally, you'll have to change this to true.